desc/compute/geom_utils.py

```diff
@@ -23,29 +23,33 @@ def reflection_matrix(normal):
     return R
 
 
-def rotation_matrix(axis, angle=None):
+def rotation_matrix(axis):
     """Matrix to rotate points about axis by given angle.
```

Comment:
need to change this, the angle is not given it is determined by axis, it reads like we are missing a parameter from this description.